draft-ietf-quic-recovery.md

```diff
@@ -1096,12 +1103,20 @@ Pseudocode for SetLossDetectionTimer follows:
 // Returns the earliest loss_time and the packet number
 // space it's from.  Returns 0 if all times are 0.
 GetEarliestLossTime():
-  time = loss_time[Initial]
+  return GetEarliestTimeAndSpace(loss_time)
+
+// Returns the earliest time_of_last_sent_ack_eliciting_packet
+// and the packet number space it's from.
+GetEarliestAckElicitingTime():
+  return GetEarliestTimeAndSpace(loss_time)
```

Comment:
You have two identical functions here that don't seem to do anything other than indirect over the global.  I don't get this change.

Reply:
I removed the functions, because they were overly verbose.  And as you said, I used the wrong argument in one of them.